src/FoldersView/FoldersListView.vala

```diff
@@ -93,6 +128,17 @@ public class Mail.FoldersListView : Gtk.Grid {
                 settings.set ("selected-folder", "(ss)", "GROUPED", grouped_folder_item.name);
             }
         });
+
+        var settings = new GLib.Settings ("io.elementary.mail");
+        settings.bind ("always-load-remote-images", load_images_menuitem, "active", SettingsBindFlags.DEFAULT);
+
+        account_settings_menuitem.clicked.connect (() => {
+            try {
+                Gtk.show_uri_on_window ((Gtk.Window) get_toplevel (), "settings://accounts/online", Gdk.CURRENT_TIME);
+            } catch (Error e) {
+                critical ("Failed to open account settings: %s", e.message);
```

Comment:
i believe we should show a dialog here, this was a direct user interaction, we shouldn't silently fail.

Reply:
Added an error dialog

![Screenshot from 2023-06-01 13 19 27](https://github.com/elementary/mail/assets/7277719/392673c6-29e6-4ca3-8539-48bab0bdebbf)